standard/expressions.md

```diff
@@ -2189,27 +2215,34 @@ When an initializer target refers to an indexer, the arguments to the indexer sh
 >
 > where `__r`, `__p1` and `__p2` are temporary variables that are otherwise invisible and inaccessible.
 >
-> If `Rectangle`’s constructor allocates the two embedded `Point` instances
+> If `Rectangle`’s constructor allocates the two embedded `Point` instances, they can be used to initialize the embedded `Point` instances instead of assigning new instances:
 >
 > ```csharp
-> public class Rectangle
+> public class Point
 > {
->     public Point P1 { get; } = new Point();
->     public Point P2 { get; } = new Point();
+>     public int X { get; set; }
+>     public int Y { get; set; }
 > }
-> ```
 >
-> the following construct can be used to initialize the embedded `Point` instances instead of assigning new instances:
+> public class Rectangle
+> {
+>     public Point P1 { get; } = new Point();
+>     public Point P2 { get; } = new Point();}
 >
-> ```csharp
-> Rectangle r = new Rectangle
+> class A
 > {
->     P1 = { X = 0, Y = 1 },
->     P2 = { X = 2, Y = 3 }
-> };
+>     static void M()
+>     {
+>         Rectangle r = new Rectangle
+>         {
+>             P1 = new Point { X = 0, Y = 1 },
```

Comment:
You've changed the code so that it doesn't demonstrate what it's meant to... these should be

```csharp
P1 = { X = 0, Y = 1 },
P2 = { X = 2, Y = 3 }
```